app.py

```diff
@@ -107,6 +107,42 @@ def add_ekubo_liquidity(
     return data
 
 
+def create_stablecoin_bundle(data: pandas.DataFrame):
+    for collateral in src.settings.COLLATERAL_TOKENS:
+        relevant_pairs = [pair for pair in data.keys() if collateral in pair and any(stablecoin in pair for stablecoin in src.settings.DEBT_TOKENS[:-1])]
+        combined_df = None
+
+        for pair in relevant_pairs:
+            df = data[pair]
+
+            if 'collateral_token_price' not in df.columns:
+                logging.warning(f"'collateral_token_price' is missing in DataFrame for pair: {pair}")
+                break
```

Comment:
if one pair doesnot have liquidable data, we still want to iterate through all other pairs. Please use `continue` instead of `break`

Reply:
Done! ✅